.drone.yml

```diff
@@ -81,6 +81,15 @@ pipeline:
       branch: [ refs/tags/* ]
       status: success
 
+  report-coverage:
+     image: robertstettner/drone-codecov
+     token: ${CODECOV_TOKEN}
+     files:
+       - h5c/vic-service/unittest-results/*.xml
```

Comment:
@mhagen-vmware We run unit test coverage for every build. And I see your only adding coverage report for our Java service portion but not including UI coverage, which can be found under `h5c/vic/src/vic-webapp/coverage`